--- vendor/pestphp/pest-plugin-type-coverage/src/TestCaseForTypeCoverage.php.bak	2026-04-06 16:00:30
+++ vendor/pestphp/pest-plugin-type-coverage/src/TestCaseForTypeCoverage.php	2026-05-05 12:12:46
@@ -113,12 +113,12 @@
                 $ruleErrors = $rule->processNode($node, $scope);
                 foreach ($ruleErrors as $ruleError) {
                     if ($this->ignored($ruleError)) {
-                        $this->ignoredErrors[] = $ruleErrorTransformer->transform($ruleError, $scope, [], $node);
+                        $this->ignoredErrors[] = $ruleErrorTransformer->transform($ruleError, $scope, $nodeType, $node->getLine());
 
                         continue;
                     }
 
-                    $actualErrors[] = $ruleErrorTransformer->transform($ruleError, $scope, [], $node);
+                    $actualErrors[] = $ruleErrorTransformer->transform($ruleError, $scope, $nodeType, $node->getLine());
                 }
             }
         }
